R/distinct.R

```diff
@@ -107,7 +107,8 @@ distinct_prepare <- function(.data,
   }
 
   # Always include grouping variables preserving input order
```

Comment:
```suggestion
  # Missing grouping variables are added to the front
```

This seems like a clearer way to talk about what this is doing, right?

I thought about suggesting you do:

```r
new_vars <- unique(c(group_vars, distinct_vars))
```

but after further review I figured out that wasn't right because you want grouping variables that also exist in `distinct_vars` to respect the order they are specified in `distinct_vars`. I think an updated comment would have gotten me there faster.